src/components/articles/ArticlesLayout.js

```diff
@@ -24,7 +25,7 @@ class ArticlesLayout extends PureComponent {
               image={logo}
               header={title}
               meta="July 2019"
-              description={`${body.slice(0, 50)}...`}
+              description={`${stripTags(body).slice(0, 50)}...`}
```

Comment:
Why did you choose to add strip tags instead of body.slice?

Reply:
I still slice the body. Since we support html now though, just splitting the body of the article would like this:
```
<b>Heading 1</b><p>This is some paragr...
```
So I just strip the html so we only get the text without the tags.